resources/views/users/view.blade.php

```diff
@@ -670,7 +669,7 @@
                     data-sort-order="asc"
                     data-sort-name="name"
                     data-toolbar="#assetsBulkEditToolbar"
-                    data-bulk-button-id="#bulkAssetEditButton"
+                    data-bulk-button-id="#bulkAssetEditButton, #checkInAll"
```

Comment:
I don't think elements can have more than one ID

Reply:
This actually worked, but it is totally unnecessary now. I was messing around with a button when I passed this id and to my delight it worked 🙂 

removed